### DISCUSSION TITLE
no alerts from Elastic agent

### DISCUSSION THREAD

**Discussion by Chizan-Dolrocks** [Security-Onion-Solutions/securityonion#11936] [2.4]
### Hello, 
I just wanted to know if I'm the only one having this problem: When I go into the Elastic Fleet manager and click on an agent to click on "View more agent metrics" The links on the left in the "Agent Health" section such as "Overview", "Agent Information", "Agent Metrics" or "Integrations" lead to a page that contains _no data_.
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/84504744/713ec29b-9120-4b04-88c4-9cf2a31b0be2)

**Thank you**

**Comment by dougburks** [comment]
@Chizan-Dolrocks Instead of creating an issue, please start a new discussion at https://securityonion.net/discuss

**Reply by TOoSmOotH** [reply]
I went ahead and converted it to a discussion.

**Comment by dougburks** [comment]
Answered the other discussion:
https://github.com/Security-Onion-Solutions/securityonion/discussions/11937#discussioncomment-7766141
